R/logo.R

```diff
@@ -5,8 +5,6 @@
 #' @param unicode Whether to use Unicode symbols. Default is `TRUE` on UTF-8
 #'   platforms.
 #'
-#' @md
-#'
```

Comment:
this is unnecessary because we set Roxygen to build the docs with Markdown globally in DESCRIPTION